src/pages/index.tsx

```diff
@@ -45,14 +54,22 @@ const IndexPage: React.FunctionComponent = () => {
       <Inner>
         <Logo />
         <Subtitle>{siteMetadata.description}</Subtitle>
-        <Link to="/blog">Blog</Link> |{' '}
+        <a
+          href="https://project-unic0rn.slack.com"
```

Comment:
update to slack invite link
```suggestion
          href="https://join.slack.com/t/project-unic0rn/shared_invite/enQtNDI1MDM2NjIxNjMyLTMwNTdmNjAyMmZhMTM1YWU0OTY2NzAyM2EwMWU1MGVlOTdmYzg5YzM3YThiMzdmZDE4NTI5MDQ3MjYxYTg4OTA"
```